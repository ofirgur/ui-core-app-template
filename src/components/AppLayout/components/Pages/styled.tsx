import styled from '@emotion/styled';

export const PagesStyled = styled.div`
  margin: 20px;
  color: #000;
`;
