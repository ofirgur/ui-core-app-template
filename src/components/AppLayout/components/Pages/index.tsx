import React from 'react';
import { Outlet } from 'react-router-dom';
import { PagesStyled } from './styled';

const Pages = () => {
  return (
    <PagesStyled>
      <Outlet />
    </PagesStyled>
  );
};

export default Pages;
