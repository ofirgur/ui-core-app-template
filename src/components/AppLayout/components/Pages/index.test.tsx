import * as React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Pages from '.';

describe('Pages', () => {
  it('should render correctly', () => {
    const { container } = render(
      <BrowserRouter>
        <Pages />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
