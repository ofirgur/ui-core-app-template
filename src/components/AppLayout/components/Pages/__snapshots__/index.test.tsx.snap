// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages should render correctly 1`] = `
<div>
  <div
    class="sc-bczRLJ igExgT"
  />
</div>
`;
